contracts/LockedFund/LockedFund.sol

```diff
@@ -308,27 +381,45 @@ contract LockedFund is ILockedFund {
 
 	/**
 	 * @notice Creates vesting contract (if it hasn't been created yet) for the calling user.
+	 * @return _vestingAddresses The New Vesting Contracts Created.
+	 * @dev Zero (0) is passed to denote all vesting for that user will be created.
+	 */
+	function createAllVesting() external returns (address[] memory _vestingAddresses) {
+		_vestingAddresses = _createVesting(msg.sender, bytes32(0));
```

Comment:
`_createVesting` func doesn't return value

Reply:
Solved in 3a11023